conda_smithy/feedstock_content/.gitignore

```
@@ -1,3 +1,13 @@
-*.pyc
+# User content belongs under recipe/.
+# Feedstock configuration goes in `conda-forge.yml`
+# Everything else is managed by the conda-smithy rerender process.
+# Please do not modify
+
+*
+!conda-forge.yml
```

Since the pattern doesn't contain a `/`, it would match in any directory.
```suggestion
!/conda-forge.yml
```